src/dlsnode/util/aio/internal/JobQueue.d

```diff
@@ -129,6 +115,68 @@ public static struct Job
     ****************************************************************/
 
     private bool is_slot_free;
+
+    /****************************************************************
+
+        Buffer for the thread to fill and to copy to the user buffer
```

Comment:
Clearer? "Buffer to be filled by the worker thread. When the job is finalised, the contents are copied into `user_buffer`."